README.md

```diff
@@ -1 +1,87 @@
+- [Working on a Technical Task Challenge](#working-on-a-technical-task-challenge)
+- [Submitting Your Technical Task Challenge](#submitting-your-technical-task-challenge)
+- [Going Further](#going-further)
+- [Honerable Mentions](#honorable-mentions)
+- [Code of Conduct](./CODE_OF_CONDUCT.md)
+
+### Adding Your Experience
+
+
+As a VC member with real-world experience, you may want to list some of your routine technical tasks. To do this, first create a fork of the repository and a new branch titled `add-to-directory`. Add your name, job title(s), years in the role(s), and tasks to the directory document. Then, make a pull request back to the main project.
+
+In doing this, you are signaling to other members that you are open for conversation about the listed tasks. If you so choose, you can post a challenge (or challenges) related to one or more of your listed tasks.
+
+**NOTE:** You **should not** post any proprietary data or company-specific tasks. Focus only on transferable tools and technology.
+
+
+### Posting a New Technical Task Challenge
+
+
+As a VC member with real-world experience, you may want to post a technical task challenge. To do this, create a fork of the repository with a new branch titled `add-<challenge-name>-challenge`. Begin by creating a new folder to hold your challenge, then add a README file. Link from the relevant task in the directory document to the `README` file in your task challenge folder.
+
+This `README` should explain the goal(s) of the task challenge and how to get started. It should also list…
+
+- the VC member who posted the task challenge (you!)
+- the challenge source/author
+- the skill level this challenge is targeting
+- any VC members currently working on the task challenge
+- any VC members who have successfully completed the task challenge, including a link to their fork of the repo
+
+**NOTE:** You do not have to create the task challenge from scratch, however, **you must have completed it before posting it** as you will be responsible for giving final feedback to VC members who attempt your challenge.
+
+#### Technical Task Challenge Skill Levels Tracks
+
+We will divide task challenges into beginner, intermediate, and advanced levels. Challenges can be multi-part, increasing in difficulty the further along you go.
+
+**Beginner level tasks:** for a person learning to code <br />
+**Intermediate level tasks:** with a focus on the working professional and building up skills <br />
+**Advanced level tasks:** for seniors who want a challenge
+
+### Working on a Technical Task Challenge
+
+As a VC member looking to build your skills, you may want to try several technical task challenges. To work on a task challenge, create your own fork of the repository and a new branch titled `add-challenge-participant`. Type your name in the appropriate spot of the README where it lists the task challenge participants, and send a pull request back to the main repo.
+
+**Who Do You Go To for Help?**
```

Comment:
**todo:** I think this is probably better as an `h3` since it directly relates to **Working on a Technical Task Challenge**

```suggestion
### Who Do You Go To for Help?
```